atomicapp/providers/openshift.py

```diff
@@ -332,10 +333,10 @@ class OpenShiftProvider(Provider):
     provider_ca = None
 
     # Parsed artifacts. Key is kind of artifacts. Value is list of artifacts.
-    openshift_artifacts = {}
+    openshift_artifacts = defaultdict(list)
 
     def init(self):
-        self.openshift_artifacts = {}
+        self.openshift_artifacts = defaultdict(list)
```

Comment:
Does it make sense why we are setting this in the class variable and the instance variable? @kadel?